tests/index.html

```diff
@@ -36,27 +36,27 @@
 
 
         // Handle extending prototypes
-        EmberENV['EXTEND_PROTOTYPES'] = !!QUnit.urlParams.extendprototypes;
+        EmberENV['EXTEND_PROTOTYPES'] = !!QUnit.urlParams.EXTEND_PROTOTYPES;
```

Comment:
The spelling differences here are because I saw no reason to manage a translation between environment variables and query params. They can all just look the same.